src/core/packet_builder.c

```diff
@@ -379,6 +379,27 @@ QuicPacketBuilderPrepare(
             Builder->Metadata->PacketId,
             Builder->BatchId);
 
+        //
+        // Occassionally skip a packet number for improved security.
```

Comment:
nit: I think it would be nice to give a bit more context.
Right now, from reading the code without a knowing what this is for, it would be hard to find how this improve security / what is the problem. Maybe having "opportunistic ack protection" somewhere would help finding relevant info?

Reply:
There's a balance here. On one hand, I agree. On the other, the more info you provide, the more info you provide to would-be attackers too. So, for now at least, I'm going to leave as is.